test/NoRecommendationProvided/cases/Under25NilmHpvCotestBefore21.yml

```diff
@@ -0,0 +1,40 @@
+---
+name: Under 25 NILM HPV Positive Cotest Before Age 21
```

Comment:
Does the cotest have to be after the patient was 21? Or just under 25? For a 20 year old patient with a NILM/HPV+ cotest, why would we want to return a null recommendation as opposed to this more specific "use clinical judgement" recommendation?

Reply:
The cotest result must be from age >= 21 to <25 to get this recommendation.

Reply:
Why is this the case? And where is this documented? Everything I see on https://jira.mitre.org/browse/CCSMCDS-6 and on https://github.com/ccsm-cds-tools/ccsm-cds-with-tests/issues/91 indicates that the patient only has to be under 25. Why does this recommendation require a patient to be older than 21, when most of the other Under 25 management recommendations (https://github.com/ccsm-cds-tools/ccsm-cds-with-tests/blob/main/cql/ManageSpecialPopulation.cql#L199) only require that the patient is < 25?

Reply:
I can't find the reference but Ginny communicated that it needed to be >= 21 since there isn't any guideline for tests before age 21. Can follow up with Ginny.

Reply:
I just saw your email. I still don't quite understand the reason for the 21 y.o. age minimum, considering the "recommendation" being given here isn't from a guideline, but is rather saying that there _is not_ a guideline recommendation for this scenario. The whole point of this update is to specify the text we give when a patient falls _outside_ of what is covered by the guideline, and as far as I am aware, the recommendation "_There is no specific recommendation for a NILM cytology and HPV + in a patient < 25 years of age. Use clinical judgement_," holds true whether or not the patient is 21+.

Regardless, the difference between this recommendation and the default one that would alternatively be given to <21 patients in this scenario is minor, since they would be told to "use clinical judgement" in either scenario, so I am fine with resolving this if we want to keep that age cutoff as-is.

Reply:
We discussed during a meeting, and have decided to proceed on with giving this recommendation text only when the cotest occurred after a patient turned 21. Patients under 21 will not receive a recommendation from this section. Thus the current testing aligns with what is expected, and this thread can be resolved.